.github/workflows/eslint.yaml

```diff
@@ -2,10 +2,11 @@ name: ESLint
 on: [pull_request]
 jobs:
   eslint:
-    name: runner / eslint
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
-      - uses: reviewdog/action-eslint@v1
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
         with:
-          reporter: github-pr-review
+          node-version: 18
```

Comment:
Any reason why you are setting node version 18 here, instead of using the same version that is used for `release.yml` (`16.x`)?

Reply:
Just checked, and our setup.ps1 scripts use node.js 18 from https://community.chocolatey.org/packages/nodejs-lts.

Reply:
I agree we should probably change the `release.yml` file then. Not sure if it should be done as part of this PR or a new one.

I'll approve this PR and let you decide where it should be updated.